test/integration/rules/link-in-text-block/link-in-text-block.html

```diff
@@ -55,7 +55,7 @@ <h1>Non-color change tests</h1>
 <p style="color: black">
   paragraph of text (pass: link is bolder than paragraph)
   <a
-    style="text-decoration: none; font-weight: bolder; color: black"
+    style="text-decoration: none; font-weight: bolder; color: #222"
```

Comment:
This would pass even without the change in style due to using the same color as the paragraph.